libs/application/templates/document-provider-onboarding/src/forms/DocumentProviderApplication.ts

```diff
@@ -0,0 +1,140 @@
+import {
+  buildForm,
+  buildMultiField,
+  buildSection,
+  buildSubSection,
+  buildIntroductionField,
+  buildTextField,
+  Form,
+  ApplicationTypes,
+} from '@island.is/application/core'
+import { m } from './messages'
+
+export const ContactInfo: Form = buildForm({
+  id: ApplicationTypes.DOCUMENT_PROVIDER_ONBOARDING,
+  ownerId: 'TODO?',
```

Comment:
If you rebase to master this attribute is no longer there

Reply:
Ahh ok... I was trying to find out what this was, but now I don't have to :)